### DISCUSSION TITLE
Unexpected token 'export' - but works locally?

### DISCUSSION THREAD

**Discussion by OmarShehata** [codesandbox/codesandbox-client#4945] [General] [answered]
This is my first sandbox, so I may be missing something obvious, but I'm trying to set up a basic app here:

https://codesandbox.io/s/stupefied-waterfall-5n9u6

This app works locally, if you download the ZIP, then run `npm install` and `npm run start`. Is this expected? What's different from the CodeSandbox environment vs locally? Is there something I can do to reproduce this error locally to try and debug it?

I found a similar bug report here, not sure if relevant: https://github.com/codesandbox/codesandbox-client/issues/4292

**Comment by siddharthkp** [comment, marked as answer]
I'm looking at the sandbox and honestly, have no idea about Cesium.

But I found this really good template by a community member (@ccszz) which can be a good starting point: https://codesandbox.io/s/cesium-tietu-eomw2

**Reply by OmarShehata** [reply]
@siddharthkp if this something that can be fixed on the library side I'm happy to look into that (I work at Cesium). I just wasn't sure how to start debugging this since it seems to be working locally. 

I do see that loading from CDN works. I wanted to see if we can get importing it as an ES6 module to work since CodeSandbox could be a great way to share examples showing how that works if we can figure this out.

Is there anything you can share about reproducing issues in the online environment locally?
